redbot/cogs/general/general.py

```diff
@@ -234,38 +240,210 @@ def __init__(self):
+                "south-korea": _("South Korea ") + "\U0001f1f0\U0001f1f7",
+            }
+            verif = {
+                "none": _("0 - None"),
+                "low": _("1 - Low"),
+                "medium": _("2 - Medium"),
+                "high": _("3 - Hard"),
+                "extreme": _("4 - Extreme"),
+            }
+
+            features = {
+                "PARTNERED": _("Partnered"),
+                "VERIFIED": _("Verified"),
+                "DISCOVERABLE": _("Server Discovery"),
+                "PUBLIC": _("Public"),
+                "INVITE_SPLASH": _("Splash Invite"),
+                "VIP_REGIONS": _("VIP Voice Servers"),
+                "VANITY_URL": _("Vanity URL"),
+                "MORE_EMOJI": _("More Emojis"),
+                "COMMERCE": _("Commerce"),
+                "LURKABLE": _("Lurkable"),
+                "NEWS": _("News Channels"),
+                "ANIMATED_ICON": _("Animated Icon"),
+                "BANNER": _("Banner Image"),
+                "MEMBER_LIST_DISABLED": _("Member list disabled"),
+            }
+            guild_features_list = [
+                f"✅ {name}" for feature, name in features.items() if feature in set(guild.features)
+            ]
+
+            joined_on = _(
+                "{bot_name} joined this server on {bot_join}. That's over {since_join} days ago!"
+            ).format(
+                bot_name=ctx.bot.user.name,
+                bot_join=guild.me.joined_at.strftime("%d %b %Y %H:%M:%S"),
+                since_join=humanize_number((ctx.message.created_at - guild.me.joined_at).days),
+            )
+
+            data = discord.Embed(description=created_at, colour=await ctx.embed_colour())
+            data.set_author(
+                name=guild.name,
+                icon_url="https://cdn.discordapp.com/emojis/457879292152381443.png"
+                if "VERIFIED" in guild.features
+                else "https://cdn.discordapp.com/emojis/508929941610430464.png"
+                if "PARTNERED" in guild.features
+                else discord.Embed.Empty,
+            )
+            data.set_thumbnail(
+                url=guild.icon_url
+                if guild.icon_url
+                else "https://cdn.discordapp.com/embed/avatars/1.png"
+            )
+            data.add_field(name=_("Members:"), value=member_msg)
+            data.add_field(
+                name=_("Channels:"),
+                value=_(
+                    "\N{SPEECH BALLOON} Text: {text}\n"
+                    "\N{SPEAKER WITH THREE SOUND WAVES} Voice: {voice}"
+                ).format(text=bold(text_channels), voice=bold(voice_channels),),
+            )
+            data.add_field(
+                name=_("Utility:"),
+                value=_(
+                    "Owner: {owner}\nRegion: {region}\nVerif. level: {verif}\nServer ID: {id}"
+                ).format(
+                    owner=bold(str(guild.owner)),
+                    region=bold(vc_regions[str(guild.region)]),
```

Comment:
In case there's gonna be added a new voice region, this will make the command less prone to errors and just show "unfriendly" name of the voice region if we don't have a friendly one in the dictionary.

I also replaced `bold()` call with wrapping in `**` as `bold()` also escapes the given text which doesn't work great with underline that is added to VIP regions.
```suggestion
                    region=f"**{vc_regions.get(str(guild.region)) or str(guild.region)}**",
```